bot.js

```diff
@@ -114,6 +114,7 @@ client.on('interactionCreate', async (interaction) => {
 
 // Start cron tasks
 weeklyLeaderboardResults(client).start();
+resetLeaderboard(client).start();
```

Comment:
Should we move these to inside the 
```js
client.on('ready', () => {...});
```
?

Reply:
Good idea, I'll look into that. Thank you.